python_modules/dagster/dagster/_core/storage/file_manager.py

```diff
@@ -221,62 +223,64 @@ def files_pipeline():
     """
     return LocalFileManager(
         base_dir=init_context.resource_config.get(
-            "base_dir", os.path.join(init_context.instance.storage_directory(), "file_manager")
+            "base_dir", os.path.join(init_context.instance.storage_directory(), "file_manager")  # type: ignore
         )
     )
 
 
-def check_file_like_obj(obj):
+def check_file_like_obj(obj: object) -> None:
     check.invariant(obj and hasattr(obj, "read") and hasattr(obj, "write"))
 
 
 class LocalFileManager(FileManager):
-    def __init__(self, base_dir):
+    def __init__(self, base_dir: str):
         self.base_dir = base_dir
         self._base_dir_ensured = False
         self._temp_file_manager = TempfileManager()
 
     @staticmethod
-    def for_instance(instance, run_id):
+    def for_instance(instance: DagsterInstance, run_id: str) -> "LocalFileManager":
         check.inst_param(instance, "instance", DagsterInstance)
         return LocalFileManager(instance.file_manager_directory(run_id))
 
-    def ensure_base_dir_exists(self):
+    def ensure_base_dir_exists(self) -> None:
         if self._base_dir_ensured:
             return
 
         mkdir_p(self.base_dir)
 
         self._base_dir_ensured = True
 
-    def copy_handle_to_local_temp(self, file_handle):
+    def copy_handle_to_local_temp(self, file_handle: FileHandle) -> str:
         check.inst_param(file_handle, "file_handle", FileHandle)
-        with self.read(file_handle, "rb") as handle_obj:
+        with self.read(file_handle, "rb") as handle_obj:  # type: ignore
             temp_file_obj = self._temp_file_manager.tempfile()
             temp_file_obj.write(handle_obj.read())
             temp_name = temp_file_obj.name
             temp_file_obj.close()
             return temp_name
 
     @contextmanager
-    def read(self, file_handle, mode="rb"):
+    def read(self, file_handle: LocalFileHandle, mode: str = "rb") -> Iterator[IOStream]:
         check.inst_param(file_handle, "file_handle", LocalFileHandle)
         check.str_param(mode, "mode")
         check.param_invariant(mode in {"r", "rb"}, "mode")
 
         encoding = None if mode == "rb" else "utf8"
         with open(file_handle.path, mode, encoding=encoding) as file_obj:
-            yield file_obj
+            yield file_obj  # type: ignore
```

Comment:
comment plz. for this file. I assume these are all related to the `self.read` calls?